libraries/AP_Scripting/examples/LED_poslight.lua

```diff
@@ -0,0 +1,71 @@
+--[[
+ Script to use LED strips as position lights.
```

Comment:
Not critical, but generally I'd recommend not doing block comments like this unless you are actually writing a block comment. The problem is we parse the script at runtime, so comments like this work to actually increase the RAM consumption while parsing a script, which is actually when you are most RAM constrained normally on a F4 based board.

Reply:
Done, thanks.